src/App.css

```diff
@@ -0,0 +1,312 @@
+.add-title {
+  color: #888;
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-family: Montserrat, sans-serif;
```

Comment:
- [ ] I can see you repeated this CSS declaration many times. I would like to advise you to put it in a **_class_** and use the **_className_** on elements. For example:
```
.montserrat {
  font-family: Montserrat, sans-serif;
}
```
Then use it on elements `<div className='montserrat'>Hello World</div>`.

Alternatively, you can use CSS variables. You can read this [article](https://www.w3schools.com/css/css3_variables.asp) to learn more about CSS variables. 👍